gm4_combat_expanded/data/gm4_combat_expanded/functions/armor/modifier/type/husk/apply.mcfunction

```diff
@@ -0,0 +1,23 @@
+# grant ramping move speed and hunger to sprinting player
+# @s = player wearing husk armour above 50% health and sprinting
+# at world spawn
+# run from armor/modifier/type/husk/check
+
+# advance sprinting timer (can advance by 1 for every husk armor piece)
+scoreboard players add @s gm4_ce_t_sprinting 1
+
+# give effect
+execute if score @s gm4_ce_t_sprinting matches 1..9 run effect give @s speed 2 0
+execute if score @s gm4_ce_t_sprinting matches 1..9 run effect give @s hunger 2 0 true
```

Comment:
These could be changed, but tbh idk if it'd be more efficient. duplicate score checks may be better than many duplicate effect calls. Only way to know would be benchmarking. It's probably fine to leave this as is.